QuestionAnswering/QA_SEQ2SEQ/data_generation/locate_then_ask/ontobuiltenv/locate/total_floor_area.py

```diff
@@ -0,0 +1,19 @@
+from constants.ontobuiltenv import OBEAttrKey
+from locate_then_ask.ontobuiltenv.locate.attr import OBEAttrLocator
+from locate_then_ask.ontobuiltenv.locate.measure import OBEOmMeasureLocator
+from locate_then_ask.ontobuiltenv.model import OBEProperty
+from locate_then_ask.query_graph import QueryGraph
+
+
+class OBETotalFloorAreaLocator(OBEAttrLocator):
+    def __init__(self):
+        self.measure_locator = OBEOmMeasureLocator()
+        
+    def locate(self, query_graph: QueryGraph, entity: OBEProperty):
+        assert entity.total_floor_area is not None
```

Comment:
See above comment.

Reply:
I've replaced the `assert` statement with `raise ValueError(...)`.